### PR TITLE
Support Steering Committee liaisons in sigs.yaml

### PR DESCRIPTION
**Pull request by nikhita**
As discussed in yesterday's (18 Jan 2020) private SC meeting ([notes](https://docs.google.com/document/d/1qazwMIHGeF3iUh5xMJIJ6PDr-S3bNkT8tNLRkSiOkOU/edit#heading=h.37nbij36msqa)).

This PR contains 3 commits:
- First commit updates the generator to support liaisons in sigs.yaml:
    - Liaisons are added under the `Contact` section.
    - Each community group README will list the SC liaison for the group.
    - `liaisons.md` will contain the map of all community groups to their liaisons (similar to [liaisons.md](https://github.com/kubernetes/steering/blob/a27fcdd82398490170e6972aca44915348f7ae60/liaisons.md) today).
- Second commit adds liaisons to sigs.yaml.
- Third commit contains autogenerated code.

/hold
/cc @kubernetes/steering-committee 
/assign @cblecker @mrbobbytables

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/lgtm

Thanks @nikhita <3

File: generator/liaisons.tmpl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Liaisons are assigned community groups at random (adjustments can be
+made, if needed) with each member having an (almost) equal distribution
+of SIGs, WGs and UGs.
+
+| Community Group            | Steering Committee Liaison |
+| -------------------------- | -------------------------- |
+{{- range .Sigs}}
+{{- if .Contact.Liaison.Name }}
+| [SIG {{.Name}}]({{.Dir}}/README.md) | {{.Contact.Liaison.Name}} |
```
Comment (spiffxp) [inline generator/liaisons.tmpl:21]: nit: same format as README's?
```suggestion
| [SIG {{.Name}}]({{.Dir}}/README.md) | {{.Contact.Liaison.Name}} (**[@{{.Contact.Liaison.GitHub}}](https://github.com/{{.Contact.Liaison.GitHub}})**) |
```

Comment (nikhita) [reply generator/liaisons.tmpl]: Done

**Comment by dims** [comment]
/lgtm

thanks @nikhita

**Comment by nikhita** [comment]
Going to cancel the hold on this. Let's merge and iterate, if needed.

/hold cancel

**Pull request merged**